rotatescreen/display_linux.py

```diff
@@ -0,0 +1,110 @@
+from Xlib import display
+from Xlib.ext import randr 
+from typing import Dict, List, Tuple
+
+# TODO: Is having these at the global level ok?
+d = display.Display()
```

Comment:
I think putting these into the `Display` class is a good idea, since it keeps them in a logical place

Reply:
I'm not sure I agree with putting these in the `Display` class, I think the crux of the issue comes down to rotate-screen's differing naming conventions with xlib. A `<rotatescreen Display>` object effectivly serves as a thin wrapper around xlib's CRTC concept, meaning all `<rotatescreen Display>`s need access to a top level `xlib.display.Display` object to function.

This effectively means the entire module has a dependency on this top level `display.Display()` object regardless of any individual `<rotatescreen Display>`. It could be possible to store these top few variables as class attributes and whilst that would work I do think semantically this inherent requirement would be a little less clear.